war/src/main/js/pluginSetupWizardGui.js

```diff
@@ -461,9 +461,34 @@ var createPluginSetupWizard = function(appendTarget) {
 			setPanel(pluginSuccessPanel, { installingPlugins : installingPlugins, failedPlugins: true });
 			return;
 		}
-
+		
+		var attachScrollEvent = function() {
+			var $c = $('.install-console-scroll');
+			if (!$c.length) {
+				setTimeout(attachScrollEvent, 50);
+				return;
+			}
+			var events = $._data($c[0], "events");
+			if (!events || !events.scroll) {
+				$c.on('scroll', function() {
```

Comment:
... for clarification: the `data` method _reads_ `data-*` attributes (see: https://api.jquery.com/data/#data-html5) it does not _write_ them, which would _not_ be considered a DOM operation.